OpenSim/Common/Exception.h

```diff
@@ -67,14 +110,31 @@ class OSIMCOMMON_API Exception  : public std::exception {
     std::string _file;
     /** Line number at which the error occurred. */
     int _line;
+    /** Delimiter/separator used to separate different parts of the message.  */
+    static const std::string _msgSeparator;
```

Comment:
> Having it there doesn't hurt.

I disagree. If we don't have a use for it now it adds complexity for no gain. Let's wait until we know whether and what customization features we'll need before implementing. This might be unneeded, or it might be insufficient. We might decide to treat the (file,line,func) boilerplate differently than the real messages, for example. Or, we might decide each level of message should be indented from the previous one. Or, maybe on Windows we should separate lines with CR/LF rather than just LF, in which case a build-time macro might be better than a static. In general it is better to start simple and add more features as they are needed.